File: dart/lib/acme/application/command/CreateUserCommand.dart
import 'package:dart/sharedkernel/application/command/command.dart';

class CreateUserCommand implements Command {
  final String _email;
  final String _password;
  final String _firstName;
  final String _lastName;

  CreateUserCommand(
      this._email, this._password, this._firstName, this._lastName);

  String get lastName => _lastName;

  String get firstName => _firstName;

  String get password => _password;

  String get email => _email;
}

File: python/src/acme/application/command/CreateUserCommand.py
import string
from python.src.sharedkernel.application.command.Command import Command


class CreateUserCommand(Command):

    def __init__(self, email: string, password: string, first_name: string, last_name: string):
        self.__last_name = last_name
        self.__first_name = first_name
        self.__password = password
        self.__email = email

    @staticmethod
    def name():
        return "CreateUserCommand"

    def email(self):
        return self.__email

    def password(self):
        return self.__password

    def first_name(self):
        return self.__first_name

    def last_name(self):
        return self.__last_name
    